haskell.nix/README.md

```diff
@@ -0,0 +1,17 @@
+Template for haskell.nix configuration for Buildkite or Gitlab CI
+
+## How to use this template
+
+1. Copy `flake.nix` file to you repository, it contains all nix dependencies and definitions.
```

Comment:
Add this to top-level flake.nix in this repo, and mention `nix flake init`

Reply:
Added